MODULE.bazel

```diff
@@ -0,0 +1,123 @@
+module(
+    name = "gcc_toolchain",
+    version = "0.8.0",
```

Comment:
I don't love that the version is hardcoded in code. I don't recall how we deal with it elsewhere.